lib/langchain/llm/ollama.rb

```diff
@@ -14,7 +14,16 @@ class Ollama < Base
       temperature: 0.8,
       completion_model_name: "llama2",
       embeddings_model_name: "llama2",
-      chat_completion_model_name: "llama2"
+      chat_completion_model_name: "llama2",
+      embedding_size: {
```

Comment:
I'm wondering whether this should exist here or as a separate constant?
```ruby
EMBEDDING_SIZES = {
  codellama: 4_096,
  ...
}
```

Reply:
Cool!